framework/src/Volo.Abp.AspNetCore.MultiTenancy/Volo/Abp/AspNetCore/MultiTenancy/AbpAspNetCoreMultiTenancyModule.cs

```diff
@@ -19,6 +19,8 @@ public override void ConfigureServices(ServiceConfigurationContext context)
                 options.TenantResolvers.Add(new HeaderTenantResolveContributor());
                 options.TenantResolvers.Add(new CookieTenantResolveContributor());
             });
+
+            context.Services.AddTransient<MultiTenancyMiddleware>();
```

Comment:
Why not implementing `ITransientDependency` instead of manual registration?

Reply:
I will update PR. 😅